.github/workflows/build-cachelib-centos.yml

```diff
@@ -8,12 +8,8 @@ jobs:
     name: "CentOS/latest - Build CacheLib with all dependencies"
     runs-on: ubuntu-latest
     # Docker container image name
-    container: "centos:latest"
+    container: "ghcr.io/igchor/cachelib-deps:centos8"
```

Comment:
Is this correct?  It looks like a personal setting infiltrating the yaml file.

Reply:
@mrambacher this is a temporary workaround - I want to change it to pmem organization but it requires the admin to make the package public (and he is on vacation right now)